akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

```diff
@@ -181,11 +181,23 @@ are specific to classic search for them in: [`akka-remote/reference.conf`](/akka
 The following documents configuration changes and behavior changes where no action is required. In some cases the old
 behavior can be restored via configuration.
 
-### Remoting dependencies have been made optional
+### Remoting
+
+#### Remoting dependencies have been made optional
 
 Classic remoting depends on Netty and Artery UDP depends on Aeron. These are now both optional dependencies that need
 to be explicitly added. See @ref[classic remoting](../remoting.md) or [artery remoting](../remoting-artery.md) for instructions.
 
+#### Remote watch and deployment have been disabled if Cluster is not used
+
+By default, these remoting features are disabled when not using akka Cluster, and behaviors
+will fall back to local. To optionally enable them without Cluster, if you understand
```

Comment:
Maybe elaborate on "fall back to local". For remote deployment it is falling back to creating a local actor. For watch it's ignoring the watch request, i.e. `Terminated` will not be delivered when the remote actor is stopped or remote node crashes.

Reply:
Elaboration added thanks.